### PR TITLE
Change integer limits and header name to use standard names.

### PR DESCRIPTION
**Pull request by BillyONeal**
1. The header name is `<limits.h>` not LIMITS.H. Use of all capital letter names to refer to headers like this is rampant in the docs, but as a non-owner of this project I don't want to attempt to fix that everywhere.  But since I'm editing this page anyway.... :)
2. We now have standard spellings for `__int64` and `unsigned __int64`. The other rows all use the standard names, and if users want what these nonstandard ones mean they can refer to `int8-int16-int32-int64.md`, so I resolved the conflict in favor of using the standard names.

### CODE REVIEW

File: docs/cpp/integer-limits.md [hunk 1]
```diff
@@ -17,37 +17,37 @@ manager: "ghogen"
...
+|**USHRT_MAX**|Maximum value for a variable of type **unsigned short**.|65535 (0xffff)|
+|**INT_MIN**|Minimum value for a variable of type `int`.|-2147483648|
+|**INT_MAX**|Maximum value for a variable of type `int`.|2147483647|
+|**UINT_MAX**|Maximum value for a variable of type `unsigned int`.|4294967295 (0xffffffff)|
+|**LONG_MIN**|Minimum value for a variable of type **long**.|-2147483648|
+|**LONG_MAX**|Maximum value for a variable of type **long**.|2147483647|
+|**ULONG_MAX**|Maximum value for a variable of type `unsigned long`.|4294967295 (0xffffffff)|
+|**LLONG_MIN**|Minimum value for a variable of type `long long`|-9223372036854775808|
+|**LLONG_MAX**|Maximum value for a variable of type `long long`|9223372036854775807|
+|**ULLONG_MAX**|Maximum value for a variable of type ``unsigned long long`|18446744073709551615 (0xffffffffffffffff)|
```
Comment (jfbastien) [inline docs/cpp/integer-limits.md]: You have one code-quote too many.

Comment (BillyONeal) [reply docs/cpp/integer-limits.md]: Fixed.

**Comment by BillyONeal** [comment]
Thanks!

**Comment by mikeblome** [comment]
I also fixed all the header names in the standard library docs. That seems to have been a convention back from when people coded with stone tools. Not sure if it's worth fixing in e.g. MFC and ATL.

**Comment by BillyONeal** [comment]
Probably not worth touching ATLMFC.

**Pull request merged**